include/zephyr/drivers/gnss.h

```diff
@@ -451,6 +456,33 @@ static inline int z_impl_gnss_get_supported_systems(const struct device *dev,
 	return api->get_supported_systems(dev, systems);
 }
 
+/**
+ * @brief Get the timestamp of the latest PPS timepulse
+ *
+ * @note PPS timepulse is considered valid when actively toggling
```

Comment:
Actively toggling what?

Reply:
The timepulse pin, it actively toggles itself.
It is valid when it is doing the thing, not valid when it isn't
![image](https://github.com/user-attachments/assets/4529df49-5538-4677-ad4d-808adf36d934)

Reply:
Alright. Let me suggest another phrasing:
```suggestion
 * @note The timestamp is considered valid when the timepulse pin is actively toggling.
```

Reply:
done